config/config-network.yaml

```diff
@@ -21,6 +21,7 @@ metadata:
     serving.knative.dev/release: devel
 
 data:
+  istio.sidecar.includeOutboundIPRanges: "172.30.0.0/16,172.20.0.0/16,10.10.10.0/24"
```

Comment:
rm?
and below

Reply:
Thanks - just saw this, should be removed!